backend/coreapp/models/preset.py

```diff
@@ -0,0 +1,21 @@
+from coreapp.models.scratch import LibrariesField
+from django.db import models
+
+
+class Preset(models.Model):
+    name = models.CharField(max_length=100)
+    platform = models.CharField(max_length=100)
+    compiler = models.CharField(max_length=100)
+    assembler_flags = models.TextField(max_length=1000, default="", blank=True)
```

Comment:
only raising for discussion, but why do we distinguish `diff_flags` differently?

Reply:
because assembler flags are passed to the assembler directly (for the left side), but diff flags are supposed to control asm-differ functionality itself, I think